src/routes/tools/local-climate-change-snapshot/_common/Header.svelte

```diff
@@ -30,8 +67,26 @@
       </Column>
     </Row>
     <Row>
-      <Column>
-        <h1>Local Climate Change Snapshot</h1>
+      <Column lg="{10}" md="{8}" sm="{4}">
+        <h1>{@html titleText}</h1>
+        {#if !showDescription}
```

Comment:
Maybe rename `showDescription` to something else like `contentOnly`? Or use the route name e.g. 

```svelte
{#if page === "explore"}
```

so that it's more obvious why something is or isn't rendered.